src/clusterfuzz/_internal/platforms/android/fetch_artifact.py

```diff
@@ -40,8 +40,8 @@ def execute_request_with_retries(request):
     try:
       result = request.execute()
       break
-    except Exception as e:
-      logs.log_error(f'Error calling endpoint {request.uri}: Error {e}')
+    except Exception:
+      pass
```

Comment:
Why are we removing this log message?

Reply:
That was a mistake on my part. I thought I had merged the upstream changes. Let me correct this.